src/gsy_e/models/market/one_sided.py

```diff
@@ -315,10 +318,9 @@ def accept_offer(  # pylint: disable=too-many-locals
 
         self.bc_interface.track_trade_event(self.time_slot, trade)
 
-        if already_tracked is False:
-            self._update_stats_after_trade(trade, offer)
-            log.info("%s[TRADE][OFFER] [%s] [%s] %s",
-                     self._debug_log_market_type_identifier, self.name, trade.time_slot, trade)
+        self._update_stats_after_trade(trade, offer)
+        log.info("%s[TRADE][OFFER] [%s] [%s] %s",
+                 self._debug_log_market_type_identifier, self.name, trade.time_slot, trade)
```

Comment:
How is the logging handled now? Will we see two lines
```
[TRADE][OFFER] ...
[TRADE][BID] ...
```
for the same trade now?

Reply:
Exactly!